pkgs/by-name/mi/micro/package.nix

```diff
@@ -93,6 +93,7 @@ let
 
     meta = {
       homepage = "https://micro-editor.github.io";
+      changelog = "https://github.com/zyedidia/micro/releases/";
```

Comment:
Technically the commit message should mention this extra line as well.

But, in practical terms, it is harmless and will not break anything per se (famous last words).